src/Model/ModelData.php

```diff
@@ -446,54 +394,31 @@ public function renderWith($template, ModelData|array|null $customFields = null)
     }
 
     /**
-     * Generate the cache name for a field
-     *
-     * @param string $fieldName Name of field
-     * @param array $arguments List of optional arguments given
-     * @return string
-     * @deprecated 5.4.0 Will be made private
+     * Get a cached value from the field cache for a field
      */
-    protected function objCacheName($fieldName, $arguments)
-    {
-        Deprecation::noticeWithNoReplacment('5.4.0', 'Will be made private');
-        return $arguments
-            ? $fieldName . ":" . var_export($arguments, true)
-            : $fieldName;
-    }
-
-    /**
-     * Get a cached value from the field cache
-     *
-     * @param string $key Cache key
-     * @return mixed
-     */
-    protected function objCacheGet($key)
+    public function objCacheGet(string $fieldName, array $arguments = []): mixed
     {
+        $key = $this->objCacheName($fieldName, $arguments);
```

Comment:
The cache name shouldn't change. It's a lot easier to reason about if we're just basing the cache name off the field name and arguments _all_ the time.
Nothing in core or supported code ever used a custom cache name, and I'd be surprised if anyone was doing it in the wild either.

Reply:
Also this and set need to be public so we can use the in `ViewLayerData`